indexing-hadoop/src/main/java/org/apache/druid/indexer/HadoopTuningConfig.java

```diff
@@ -372,6 +381,7 @@ public HadoopTuningConfig withWorkingPath(String path)
         appendableIndexSpec,
         maxRowsInMemory,
         maxBytesInMemory,
+        false,
```

Comment:
typo? this will turn `useMaxMemoryEstimates` off irrespectively its current setting 
```suggestion
        useMaxMemoryEstimates,
```